src/genie/libs/parser/iosxe/show_interface.py

```diff
@@ -1571,6 +1571,87 @@ def cli(self, ip,output=None):
         return interface_dict
 
 
+class ShowIpv6InterfaceBriefPipeIpSchema(MetaParser):
+    """Schema for show ipv6 interface brief | include <WORD>"""
+    schema = {'interface':
+                {Any():
+                    {Optional('ipv6_address'): str,
+                    Optional('interface_status'): str,
+                    Optional('protocol_status'): str}
+                },
+            }
+
+
+class ShowIpv6InterfaceBriefPipeIp(ShowIpv6InterfaceBriefPipeIpSchema):
+    """Parser for:
+     show ipv6 interface brief | include <WORD>
+     parser class implements detail parsing mechanisms for cli
+
+     In reality, the 'include' would only show the IPv6 address without
+     interface information. This class mimics the IPv4 behavior by executing
+
+     show ipv6 interface brief
+
+     And parsing the address <> interface name mapping to generate a similar
+     output. Also note that because link-local addresses may be statically
+     defined for interfaces, and every interface can have the same LL
+     address, this implementation supports parsing for fe80:: addresses
+     and will return every matching interface.
+    """
+    #*************************
+    # schema - class variable
+    #
+    # Purpose is to make sure the parser always return the output
+    # (nested dict) that has the same data structure across all supported
+    # parsing mechanisms (cli(), yang(), xml()).
+
+    cli_command = 'show ipv6 interface brief'
```

Comment:
The cli command does not match the documented command with the filter. The command should match the documentation otherwise its confusing for users.

Reply:
Updated the cli_command and verified tests pass as expected

Reply:
Update - although tests pass, expected implementation does not work on live devices (as should have been expected) as the ```| include``` will only return a v6 address and not the interface information. Will work with you on a proposed solution or to determine if there is a more appropriate method to accomplish the intended use case for this parser.